<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../usr/share/icons/gnome/32x32/mimetypes/image.png</normaloff>../../../../../../usr/share/icons/gnome/32x32/mimetypes/image.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select directory</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Format:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConvert">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Image(s)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
